### ISSUE TITLE
how to deal with the negative samples

### ISSUE THREAD

**Issue by xmaj2008** [qqwweee/keras-yolo3#294]
in the official version of darknet ,the author give  the following solution to do with the negative samples:

desirable that your training dataset include images with non-labeled objects that you do not want to detect - negative samples without bounded box (empty .txt files) - use as many images of negative samples as there are images with objects

if i wanna use the negative samples with this keras-version like the darknet, how can i achieve it.

**Comment by michalkowalski94** [comment]
Anyone?

**Comment by YourGc** [comment]
Do you mean put the negative samples into the annotation file?(txt file) and without boxes or classes?

**Comment by Hansmu** [comment]
I think that's what the OP meant. At least that's what brought me here - wanting to add pictures with none of my classes on them to the training set. @xmaj2008 @michalkowalski94 @YourGc Have any of you figured out how to do that?

**Comment by YourGc** [comment]
> I think that's what the OP meant. At least that's what brought me here - wanting to add pictures with none of my classes on them to the training set. @xmaj2008 @michalkowalski94 @YourGc Have any of you figured out how to do that?

Just add **Image name** into annotation file  but with **no coordinates or labels**

**Comment by Hansmu** [comment]
> > I think that's what the OP meant. At least that's what brought me here - wanting to add pictures with none of my classes on them to the training set. @xmaj2008 @michalkowalski94 @YourGc Have any of you figured out how to do that?
> 
> Just add **Image name** into annotation file but with **no coordinates or labels**

Seems like that did the trick, cheers

**Comment by xmaj2008** [comment]
> > I think that's what the OP meant. At least that's what brought me here - wanting to add pictures with none of my classes on them to the training set. @xmaj2008 @michalkowalski94 @YourGc Have any of you figured out how to do that?
> 
> Just add **Image name** into annotation file but with **no coordinates or labels**

that's it,thanks~

**Comment by WilburZjh** [comment]
> > I think that's what the OP meant. At least that's what brought me here - wanting to add pictures with none of my classes on them to the training set. @xmaj2008 @michalkowalski94 @YourGc Have any of you figured out how to do that?
> 
> Just add **Image name** into annotation file but with **no coordinates or labels**

Hi @YourGc , I was wondering how to add Image name into the txt file? Since for yolo, the annotation is a txt file, could you give me some more suggestions?
